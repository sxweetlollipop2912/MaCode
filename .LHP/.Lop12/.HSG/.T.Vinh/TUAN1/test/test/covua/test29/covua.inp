30
x0d0ddd.10d11d001001d0d0.11d10
1x1011d1d11d.00dd1dd00100dd101
d0x0dd.010d110000ddd0d001d101d
111x01100d01.0.00d0dd01100110d
d0d1xd01d1000d010.10.010010d.1
d0d0dx00dd0d101dd01100ddddd1d0
dd.011x10d1d1d1ddd0110d1dddd01
.011010x0d01000011dd1000110100
0d01dd11x00dd0dd1d0d00ddd01011
101d0ddd1x1100d111d11010dd11d1
d0d1110110x1d.d0001111d1011d11
0d001dd0d00x11100010d111d00110
0.0.1001d1d0xd1ddd100100ddd001
d111d1d111.0dx10010d00d0010110
111.1001ddd000x01101dd1d001010
1d110dd1d011d11xd110011dd1d10d
0d111dd00011d10dxdd1.dd0000000
10dd.1d0d011d000dx0111111d1010
1dd1001d1d000110d1xd0dd0dd0100
0ddd100dd0011d0100dxdd01100d11
d11d.100100d11d1.01dx000d011d1
11d11111110001d0d0dd1x11.1010d
d0100dd1d0d01d00d0d110xddd011d
11101d01d10011dd101010dx00.010
.1011dd0dd1dd11d10d0d.d1xd100.
0dd10dd01d01d0101dd110d1dx0ddd
0d001dd10001d10d1011011.01x001
d010d0d010d01010110d00011d1x01
0101.d110d0010011010d1001d11xd
10dd01010001011d11100dd1.d00dx