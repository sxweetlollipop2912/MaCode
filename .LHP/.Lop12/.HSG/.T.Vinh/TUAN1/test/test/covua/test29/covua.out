11 16 22
